scripts/inner/kubestellar-kube-bind

```diff
@@ -89,8 +89,11 @@ else
     fi
 fi
 
+request_file=$kb_dir/$resource_to_bind-request.yaml
+trap 'rm $request_file' EXIT
+
 echo "binding $resource_to_bind for $space_name"
-if ! result="$(kubectl bind http://127.0.0.1:8080/export --skip-konnector --unattended --resource $resource_to_bind 2>&1)"; then
+if ! result="$(kubectl bind http://127.0.0.1:8080/export --dry-run --skip-konnector --unattended --resource $resource_to_bind 2>&1 1>$request_file)"; then
```

Comment:
```
bash-3.2$ result="$(date 2>&1 1>datefile)"
bash-3.2$ echo "result=<$result>"
result=<>
```

Reply:
So this works because `kubectl bind` sends the sought output to stderr?

Reply:
Yes!